components/service/telemetry/README.md

```diff
@@ -2,6 +2,12 @@
 
 A generic library for sending telemetry pings from Android applications to Mozilla's telemetry service.
 
+## Deprecated
+This component is now deprecated: no further updates or fixes will be provided.
```

Comment:
nit: I'll provide a fix if something blows up completely. But I know what you mean. 🙃

Reply:
Hah, good point, yes. Let me rephrase this to be less scary and surprising!